services/conservation_portfolios/src/util/error_handler.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger, EHFunction } from './types';
+
+/**
+ * Wrapper to catch errors from every endpoint. When an error occurs in a lower layer,
+ * depending on the error it is caught and organized to be thrown again an then log it here,
+ * or it's just handled here.
+ *
+ * @param {Object} logger logger instance.
+ * @param {Function} callback the function to execute.
+ *
+ * @returns {Function} router function to continue restify workflow
+ */
+export default (logger: Logger): EHFunction =>
+  (callback) =>
+  async (req, res, next) => {
+    logger.info(`[${req.id()}] received - ${req.href()}`);
+    try {
+      await callback(req, res, next);
```

Comment:
Here, vs code detects await as useless in this expression. ` 'await' has no effect on the type of this expression.` I think this is related to something missing in the API doc or missing the promise anotation.

Reply:
The problem was with the callback function. Apidoc is only related with generated documentation for endpoints.

Reply:
ok